site/src/components/pages/user/UserPageComponent.tsx

```diff
@@ -101,127 +88,36 @@ export const UserPageComponent: VoidFunctionComponent<UserPageProps> = ({
-                rowSpacing={{ xs: 2, sm: 4 }}
-                container
-              >
-                {blocks
-                  .slice(0, 4)
-                  .map(
-                    (
-                      {
-                        displayName,
-                        description,
-                        icon,
-                        lastUpdated,
-                        version,
-                        name,
-                        image,
-                        blockPackagePath,
-                      },
-                      index,
-                    ) => (
-                      <Grid key={name} item xs={12} md={6}>
-                        <OverviewCard
-                          url={blockPackagePath}
-                          description={description!}
-                          icon={icon}
-                          image={image}
-                          lastUpdated={lastUpdated}
-                          title={displayName!}
-                          type="block"
-                          version={version}
-                          // we only show images for the first 2 blocks
-                          // on desktop
-                          hideImage={index > 1 || isMobile}
-                        />
-                      </Grid>
-                    ),
-                  )}
-                {entityTypes.map(({ entityTypeId, schema, updatedAt }) => (
-                  <Grid key={entityTypeId} item xs={12} md={6}>
-                    <OverviewCard
-                      url={schema.$id}
-                      description={schema.description as string}
-                      lastUpdated={
-                        typeof updatedAt === "string"
-                          ? updatedAt
-                          : updatedAt?.toISOString()
-                      } // temporary hack to stop type error
-                      title={schema.title}
-                      type="schema"
-                    />
-                  </Grid>
-                ))}
-              </Grid>
+              <TabPanelContentsWithOverview
+                user={user}
+                blocks={blocks}
+                entityTypes={entityTypes}
+              />
             </TabPanel>
             <TabPanel activeTab={activeTab} value="blocks" index={1}>
-              {blocks.map(
-                ({
-                  displayName,
-                  description,
-                  icon,
-                  lastUpdated,
-                  name,
-                  blockPackagePath,
-                }) => (
-                  <ListViewCard
-                    key={name}
-                    type="block"
-                    icon={icon}
-                    title={displayName!}
-                    description={description}
-                    lastUpdated={lastUpdated}
-                    url={blockPackagePath}
-                  />
-                ),
-              )}
+              <TabPanelContentsWithBlocks //
```

Comment:
```suggestion
              <TabPanelContentsWithBlocks 
```

Reply:
Done in https://github.com/blockprotocol/blockprotocol/pull/255/commits/9628342573958d41dc03c31c7c672bfd0bac5477. I originally added `//` to make sections look more look-alike. But not worth it if this is confusing.